Tasks/DownloadPackageV1/main.ts

```diff
@@ -74,8 +74,9 @@ async function main(): Promise<void> {
             tl.debug("Trying to resolve latest version for " + packageId);
             version = await p.resolveLatestVersion(feed.feedId, feed.projectId, packageId);
             tl.debug("Resolved version to: " + version);
+            console.log(tl.loc("Info_ResolveLatestVersion",version));
```

Comment:
Suggestion: move this down two lines (so it prints unconditionally) and make it "Downloading {packageName} {packageVersion}" (or similar)

Reply:
we dont have the package name here - only the id. 
I initially tried this:
ex
![image](https://user-images.githubusercontent.com/53092095/217111704-14ca86d4-fa2c-4c3c-9f73-19476b822887.png)
 
but thought it was not super readable.

Reply:
oh, gross. never mind then.